rplugin/node/vim-package-info/parser.js

```diff
@@ -78,7 +83,11 @@ function getVersion(data, depSelector, dep, line) {
 
   const verinfo = data[depSelector][dep];
 
-  if (typeof verinfo === "string") return verinfo;
+  if (typeof verinfo === "string") {
+    // ensure we only return version info (e.g. ">=1.0" -> "1.0")
+    return verinfo.match(/(\d+\.)?(\d+\.)?(\*|\d+)/)[0];
```

Comment:
I guess we could return the whole thing here and just strip out the extra stuff in `diff.js`.

In the future I am planning to add support for finding the latest version you can update based on the current version. Things like specifiers for stick to current major version and stuff like that.

If you see we are actually doing something similar right now in `diff.js`, well just removing the first char. But I guess you get the idea.

So could you change the stripping of `>=` to `diff.js`